spec/acceptance/rails_middleware_spec.rb

```diff
@@ -16,5 +16,17 @@
       @app.initialize!
       assert @app.middleware.include?(Rack::Attack)
     end
+
+    it "can be configured via a block" do
```

Comment:
I just tried [if this test file was even running](https://github.com/fatkodima/rack-attack/compare/multiple-instances...santib:multiple-instances-santib?expand=1) and the [CI passed](https://github.com/santib/rack-attack/actions/runs/7476191579)..

I'm working on a fix

Reply:
https://github.com/rack/rack-attack/pull/649

Reply:
when I tried rebasing on that branch so that this test file is run, the test fails

```
$ BUNDLE_GEMFILE=gemfiles/rails_7_1.gemfile be ruby -Itest spec/acceptance/rails_middleware_spec.rb

.F

Failure:
Middleware for Rails#test_0002_can be configured via a block [spec/acceptance/rails_middleware_spec.rb:26]:
Expected: 403
  Actual: 200

bin/rails test spec/acceptance/rails_middleware_spec.rb:20
```